src/frontend/packages/lib_video/src/api/thumbnailUploadEnded/index.spec.ts

```diff
@@ -0,0 +1,85 @@
+import fetchMock from 'fetch-mock';
+import {
+  modelName,
+  thumbnailMockFactory,
+  uploadState,
+  useJwt,
+  videoMockFactory,
+} from 'lib-components';
+
+import { thumbnailUploadEnded } from '.';
+
+describe('sideEffects/thumbnailUploadEnded', () => {
+  beforeEach(() => {
+    useJwt.getState().setJwt('some token');
+  });
+
+  afterEach(() => fetchMock.restore());
+
+  const video = videoMockFactory({
+    upload_state: uploadState.READY,
+  });
+
+  const thumbnail = thumbnailMockFactory({
+    upload_state: uploadState.READY,
```

Comment:
Same, is it necessary to init `upload_state`? I can see you don't use it in your tests.